lib/uniris/oracles/oracles/uco_price.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Uniris.Oracles.UcoPrice do
+  @moduledoc false
+
+  alias Uniris.Oracles.Coingecko
+
+  @oracles [Coingecko]
```

Comment:
Maybe rename it as  `endpoints` or `sources` as Coingecko is more an external source instead of an Oracle